src/serena/cli.py

```diff
@@ -7,6 +7,13 @@
 from pathlib import Path
 from typing import Any, Literal
 
+# Import CREATE_NO_WINDOW safely across platforms
+try:
+    from subprocess import CREATE_NO_WINDOW
```

Comment:
import this from util.shell pls, then you don't need the try-except in each module